content/tutorial/tutorial.md

```diff
@@ -581,19 +588,19 @@ function Square(props) {
 }
 ```
 
-We have changed `this.props` to `props` both times it appears.
+हमने `this.props` को `props` में बदल दिया है, दोनों जगह जहाँ इसका इस्तेमाल हो रहा था।
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/QvvJOv?editors=0010)**
+**[अभी तक का पूरा कोड देखें](https://codepen.io/gaearon/pen/QvvJOv?editors=0010)**
 
->Note
+>ध्यान दें
 >
->When we modified the Square to be a function component, we also changed `onClick={() => this.props.onClick()}` to a shorter `onClick={props.onClick}` (note the lack of parentheses on *both* sides).
+>जब हमने Square को फ़ंक्शन कौम्पोनॅन्ट होने के लिए संशोधित किया, तो हमने `onClick = {() => this.props.onClick ()}` को एक छोटे `onClick = {props.onClick}` में बदल दिया (नोट पर कोष्ठक की कमी पर ध्यान दें) *दोनों पक्षों)। एक क्लास में, हमने सही `this` मान को एक्सेस करने के लिए एक एरो फ़ंक्शन का उपयोग किया, लेकिन एक फ़ंक्शन कौम्पोनॅन्ट में हमें `this` के बारे में चिंता करने की आवश्यकता नहीं है।
```

Comment:
Some places Square has been translated to Hindi, some it has not been, can we be consistent?

Reply:
में बदल दिया tha

Reply:
(नोट पर कोष्ठक की कमी पर ध्यान दें) *दोनों पक्षों)। एक क्लास में, हमने सही `this` मान को एक्सेस करने के लिए एक एरो फ़ंक्शन का उपयोग किया, लेकिन एक फ़ंक्शन कौम्पोनॅन्ट में हमें `this` के बारे में चिंता करने की आवश्यकता नहीं है। -> revisit?

Reply:
> Some places Square has been translated to Hindi, some it has not been, can we be consistent?

whereever Square is a component its not translated, and if its not referenced as a component there I have translated

Reply:
> नोट पर कोष्ठक की

```
(फंक्शन के *दोनों तरफ* पैरेंथेसेस ना होने पर ध्यान दें) ? 
```

Reply:
works